pyogrio/_ogr.pxd

```diff
@@ -240,22 +240,25 @@ cdef extern from "ogr_api.h":
 
     void            OGR_Fld_SetSubType(OGRFieldDefnH fielddefn, OGRFieldSubType subtype)
 
-    OGRGeometryH    OGR_G_CreateGeometry(int wkbtypecode)
-    void            OGR_G_DestroyGeometry(OGRGeometryH geometry)
-    void            OGR_G_ExportToWkb(OGRGeometryH geometry, int endianness, unsigned char *buffer)
-    void            OGR_G_GetEnvelope(OGRGeometryH geometry, OGREnvelope* envelope)
-    OGRErr          OGR_G_ImportFromWkb(OGRGeometryH geometry, const void *bytes, int nbytes)
-    int             OGR_G_IsMeasured(OGRGeometryH geometry)
-    void            OGR_G_SetMeasured(OGRGeometryH geometry, int isMeasured)
-    int             OGR_G_Is3D(OGRGeometryH geometry)
-    void            OGR_G_Set3D(OGRGeometryH geometry, int is3D)
-    int             OGR_G_WkbSize(OGRGeometryH geometry)
-    OGRGeometryH    OGR_G_ForceToMultiPoint(OGRGeometryH geometry)
-    OGRGeometryH    OGR_G_ForceToMultiLineString(OGRGeometryH geometry)
-    OGRGeometryH    OGR_G_ForceToMultiPolygon(OGRGeometryH geometry)
+    OGRGeometryH        OGR_G_CreateGeometry(int wkbtypecode)
```

Comment:
Most of this diff is a white space change to handle consistent indents; additions are `OGR_G_GetGeometryType`, `OGR_G_GetLinearGeometry`, `OGR_GT_IsNonLinear`